subsys/bluetooth/host/att.c

```diff
@@ -179,6 +178,10 @@ void att_sent(struct bt_conn *conn, void *user_data)
 	}
 }
 
+#if defined(CONFIG_BT_TESTING)
+atomic_t flag_send_over_eatt = (atomic_t)false;
+#endif
```

Comment:
This is the first time we've introduced something like this. 

I am not saying it isn't a good way of doing it, but we may need to gather more input on whether we want to achieve it this way. 

The alternatives are:
1) Add a function that returns this value (perhaps for a given `bt_conn` object?).
2) Instead of testing this flag in a babblesim, a simple `__ASSERT` would provide the same information, would it not?
3) Testing whether a request was _received_ on a EATT bearer instead.

Reply:
yes, they are valid option. We need to decide then which one is the best for our case.